### ISSUE TITLE
Finland is missing from big-mac-full-index.csv

### ISSUE THREAD

**Issue by bebraw** [TheEconomist/big-mac-data#24]
I see Finland in your source data, however (maybe missing data for the latest year or something like that?).

**Comment by bebraw** [comment]
I noticed Estonia is another one.

I also know why. It's due to https://github.com/TheEconomist/big-mac-data/blob/master/data-generator.R missing both from the `big_mac_countries` array for some reason. I see they are in `regression_countries`, however.

**Comment by houmie** [comment]
You are right, that's really strange. I wonder if this can be fixed by adding them manually, but the data must still be provided from somewhere...
